benchmarks/renderImageItem.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+import pyqtgraph as pg
+
+try:
+    import cupy as cp
+
+    pg.setConfigOption("useCupy", True)
+except ImportError:
+    cp = None
+
+try:
+    import numba
+except ImportError:
+    numba = None
+
+def renderQImage(*args, **kwargs):
+    imgitem = pg.ImageItem(axisOrder='row-major')
+    if 'autoLevels' not in kwargs:
+        kwargs['autoLevels'] = False
+    imgitem.setImage(*args, **kwargs)
+    imgitem.render()
+
+
+def prime_numba():
+    shape = (64, 64)
+    lut_small = np.random.randint(256, size=(256,3), dtype=np.uint8)
+    lut_big = np.random.randint(256, size=(512,3), dtype=np.uint8)
+    for lut in [lut_small, lut_big]:
+        renderQImage(np.zeros(shape, dtype=np.uint8), levels=(20,220), lut=lut)
+        renderQImage(np.zeros(shape, dtype=np.uint16), levels=(250,3000), lut=lut)
+        renderQImage(np.zeros(shape, dtype=np.float64), levels=(-4.0,4.0), lut=lut)
+
+
+class _TimeSuite(object):
+    def __init__(self):
+        super(_TimeSuite, self).__init__()
+        self.float_data = None
+        self.uint8_data = None
+        self.uint8_lut = None
+        self.uint16_data = None
+        self.uint16_lut = None
+
+    def setup(self):
+        size = (self.size, self.size)
+        self.float_data, self.uint16_data, self.uint8_data, self.uint16_lut, self.uint8_lut = self._create_data(
+            size, np
+        )
+        if numba is not None:
+            # ensure JIT compilation
+            pg.setConfigOption("useNumba", True)
+            prime_numba()
+            pg.setConfigOption("useNumba", False)
+        if cp:
+            renderQimage(cp.asarray(self.uint16_data["data"]))  # prime the gpu
```

Comment:
typo. should be `renderQImage`

Reply:
Fixed. I did run VideoSpeedTest.py on an old machine with cupy but not the asv benchmark.